.github/workflows/terraform-apply.yml

```diff
@@ -0,0 +1,42 @@
+name: Terraform apply
+
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - 'config/clusters/**'
+
+defaults:
+  run:
+    working-directory: config/clusters
+
+env:
```

Comment:
I think we should just configure aws credentials like we do in master CI: https://github.com/falcosecurity/test-infra/blob/master/.github/workflows/master.yml#L21

Reply:
I totally agree to follow the best practice to assume a role. Thanks @FedeDP

Reply:
Implemented @FedeDP